docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -3282,6 +3282,7 @@ The result will be an RpcResponse JSON object with `value` set to a JSON object
     - `data: <[string, encoding]|object>`, data associated with the account, either as encoded binary data or JSON format `{<program>: <state>}`, depending on encoding parameter
     - `executable: <bool>`, boolean indicating if the account contains a program \(and is strictly read-only\)
     - `rentEpoch: <u64>`, the epoch at which this account will next owe rent, as u64
+- `unitsConsumed: <u64 | null>`, The number of compute budget units consumed during the processing of this transaction
```

Comment:
nit: "undefined" right?  (older RPC nodes just won't include this field at all, which is expressed by "undefined" instead of "null")

Reply:
Sorry, that's correct. Should be `undefined`